compiler/kernel/expr.go

```diff
@@ -317,7 +317,7 @@ func compileCall(zctx *zed.Context, call dag.Call) (expr.Evaluator, error) {
 			return nil, fmt.Errorf("has(): bad argument: %w", err)
 		}
 		return expr.NewHas(exprs), nil
-	case call.Name == "unflatten":
+	case call.Name == "zeek":
```

Comment:
We're looking for a different name for this, right?

Reply:
Oh yeah I'll change that